code/modules/clothing/clothing_accessories.dm

```diff
@@ -124,28 +124,39 @@
 	set name = "Remove Accessory"
 	set category = "Object"
 	set src in usr
-	if(!isliving(usr))
+
+	remove_accessory(usr, pick_accessory_to_remove(usr, usr))
+	if(!LAZYLEN(accessories))
+		verbs -= /obj/item/clothing/proc/removetie_verb
```

Comment:
The verb removal probably ought to just be `remove_accessory`; but it also has a flaw as is.

So not only are you not performing this if we are stripping an accessory via `/datum/strippable_item/mob_item_slot/jumpsuit/alternate_action(atom/source, mob/user)` but the verb is only added if the accessory is removeable. So it would be more correct to remove the verb in `remove_accessory` if `accessories` is empty, or has no accessories that have `removable` true (I don't think we currently have any that aren't removable though but we might as well support the functionality since it was bothered to be implemented at least partially).

![image](https://github.com/cmss13-devs/cmss13/assets/76988376/b882260f-ae5f-4ea0-9fac-8a9c7a40ec28)